### ISSUE TITLE
gRPC Tracing Errors

### ISSUE THREAD

**Issue by tparikh** [signalfx/splunk-otel-java#232]
I am using the latest version of splunk-otel-javaagent and we are seeing following errors in logs which is preventing the application from being properly configured and starting up

```
[opentelemetry.auto.trace 2021-05-14 14:29:06:813 +0000] [grpc-default-executor-0] ERROR io.opentelemetry.javaagent.shaded.instrumentation.grpc.v1_5.internal.ContextStorageBridge - Context was not attached when detaching                
java.lang.Throwable                                                                                                                                                                                                                         
        at io.opentelemetry.javaagent.shaded.instrumentation.grpc.v1_5.internal.ContextStorageBridge.detach(ContextStorageBridge.java:86) 
```

Upstream has removed the check and log message in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2970/files.

**Comment by mateuszrzeszutek** [comment]
Hey @tparikh,

This was fixed in the upstream 1.2.0 version, which was released last Friday - you can expect our release (containing all fixes from OTel instrumentation 1.2.0) today or tomorrow.

**Comment by mateuszrzeszutek** [comment]
Hey @tparikh,

We've just released [v0.11.0](https://github.com/signalfx/splunk-otel-java/releases/tag/v0.11.0) yesterday - this release contains the gRPC bridging fixes. Please try it out and check if it fixes your problem.

**Comment by laurit** [comment]
@tparikh let us know if this is still an issue. Closing as we believe it should have be fixed.
